src/query/sql/src/planner/binder/table.rs

```diff
@@ -1010,17 +1010,19 @@ impl Binder {
     }
 
     #[async_backtrace::framed]
-    pub(crate) async fn resolve_data_source(
+    pub async fn resolve_data_source(
         &self,
-        tenant: &str,
+        _tenant: &str,
         catalog_name: &str,
         database_name: &str,
         table_name: &str,
         travel_point: &Option<NavigationPoint>,
     ) -> Result<Arc<dyn Table>> {
-        // Resolve table with catalog
-        let catalog = self.catalogs.get_catalog(tenant, catalog_name).await?;
-        let mut table_meta = catalog.get_table(tenant, database_name, table_name).await?;
+        // Resolve table with ctx
```

Comment:
Here need add more comments to desp why we change to ctx.get_table

Reply:
ok